galerautils/src/gu_config.hpp

```diff
@@ -84,7 +84,7 @@ class gu::Config
         else
         {
 #ifndef NDEBUG
-            log_error << key << " not recognized.";
+            log_debug << "Key '" << key << "' not recognized.";
 #endif
```

Comment:
If this logging is changed to debug, will there be any indication about which key is not recognized?